hypothesis-python/tests/cover/test_settings.py

```diff
@@ -469,7 +469,7 @@ def test_show_changed():
 
 
 def test_note_deprecation_checks_date():
-    with pytest.warns(None) as rec:
+    with pytest.warns() as rec:
```

Comment:
```suggestion
    with pytest.warns(HypothesisDeprecationWarning) as rec:
```

(and how ironic that I wrote the deprecation for `pytest.warns(None)`, and didn't clean this up myself)